appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -164,8 +166,7 @@
     private boolean outputLauncher;
     private File copyDirectory;
     private Properties userSystemProperties;
-    private List<String> repositoryURLs;
-    private final String defaultMavenRepository = "https://repo.maven.apache.org/maven2/";
+    private List<String> repositoryURIs;
```

Comment:
Recipe for an NPE. Should be initialized right here and made final

Reply:
Maybe better in constructor?

Reply:
I wouldn't create a constructor just for this, no :)

Reply:
Private constructor already found

Reply:
oh yeah all the way down there... never mind :)